pyth_observer/calendar.py

```diff
@@ -25,9 +25,9 @@
 
 
 class HolidayCalendar():
-    def is_market_open(self, product, dt):
+    def is_market_open(self, asset_type, dt):
```

Comment:
the reason for that is because we are currently only checking the `asset_type` in the function and the `product` var from PriceAccountValidationEvent and PriceAccountValidationEvent had different structure

Reply:
for e.g. for `PriceValidationEvent` it's `self.price.product_attrs['asset_type']` and for `PriceAccountValidationEvent` it's `self.price_account.product.attrs['asset_type']`